src/content/docs/bff/fundamentals/multi-frontend/index.mdx

```diff
@@ -143,26 +151,71 @@ A frontend can define its own API surface, by specifying remote APIs.
 var frontend = new BffFrontend(BffFrontendName.Parse("frontend1"))
    .WithRemoteApis(
       // map the local path /path to the remote api
-      new RemoteApi(LocalPath.Parse("/some_path"), new Uri("https://remote-api")))
+      new RemoteApi("/some_path", new Uri("https://remote-api")))
 
       // You can also configure various options, such as the type of token,
       // retrieval parameters, etc.. 
-      new RemoteApi(LocalPath.Parse("/with_options"), new Uri("https://remote-api")))
+      new RemoteApi("/with_options", new Uri("https://remote-api")))
         .WithAccessToken(RequiredTokenType.UserOrClient),
         .WithAccessTokenRetriever<ImpersonationAccessTokenRetriever>(),
         .WithUserAccessTokenParameters(new BffUserAccessTokenParameters { Resource = Resource.Parse("urn:isolated-api") }));
 ```
 
 See the topic on [Token Management](/bff/fundamentals/tokens.md) for more information about the various token management options. 
 
-## Index HTML Proxying
+## Handling SPA static assets
+
+The BFF can be configured to handle the static file assets that are typically used when developing SPA based apps. 
+
+### Proxying the Index.HTML only
 
 When deploying a multi-frontend BFF, it makes most sense to have the frontend's configured with an `index.html` file that is retrieved from a Content Delivery Network (CDN). 
 
+This can be done in various ways. For example, if you use Vite, you can publish static assets with a 'base' configured. This will make sure that any static asset,
+(such as images, scripts, etc) are retrieved directly from the CDN. This gives the best performance. 
```

Comment:
```suggestion
This can be done in various ways. For example, if you use Vite, you can publish static assets with a base URL configured. This will make sure that any static asset, (such as images, scripts, etc) are retrieved directly from the CDN for best performance.
```